k4e-agent-sti.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+set -e
+
+# The scripts performs the following steps on rhel-8.5:
+# * Install imagebuilder 
+# * Create rpms for yggdrasil and k4e-device-worker
+# * Publish rpms as yum repo to be consumed by imagebuilder
+# * Publish aarch64 rhel-image for edge-device
+
+# The script assumes the machine is registered via subscription-manager
+if ! subscription-manager refresh; then
+   echo  "The machine is not registered."
+   exit 1
+fi
+
+#---------------------------
+# Install osbuild components
+#---------------------------
+dnf install -y osbuild-composer composer-cli cockpit-composer bash-completion
+systemctl enable osbuild-composer.socket --now
+systemctl enable cockpit.socket --now
+systemctl start firewalld
+firewall-cmd --add-service=cockpit && firewall-cmd --add-service=cockpit --permanent
+source  /etc/bash_completion.d/composer-cli
+
+#-----------------------------------
+# Build packages for k4e from source
+#-----------------------------------
+mkdir /home/builder
+
+dnf install -y dbus-devel systemd-devel git golang rpm-build
+rm -rf ~/rpmbuild/rpmbuild/*
+
+# Build yggdrasil rpm
+git clone https://github.com/jakub-dzon/yggdrasil.git /home/builder/yggdrasil
+cd /home/builder/yggdrasil
+export CGO_ENABLED=0
+
+export ARCH=$(uname -i)
+GOPROXY=proxy.golang.org,direct PWD=$PWD spec=$PWD outdir=$PWD make -f .copr/Makefile srpm
+rpm -ihv $(ls -ltr yggdrasil-*.src.rpm | tail -n 1 | awk '{print $NF}')
+if [ "$ARCH" = "aarc64" ]; then
+    # Turn ELF binary stripping off in %post
+    sed -i '1s/^/%global __os_install_post %{nil}/' ~/rpmbuild/SPECS/yggdrasil.spec
+fi
+rpmbuild -bb ~/rpmbuild/SPECS/yggdrasil.spec --target $ARCH
+
+# Build k4e-device-worker rpm
+git clone https://github.com/jakub-dzon/k4e-device-worker.git /home/builder/k4e-device-worker
```

Comment:
When I built `k4e-device-worker` on rhel8.4 I had to install additional packages:

dnf install http://mirror.centos.org/centos/8/PowerTools/x86_64/os/Packages/gpgme-devel-1.13.1-7.el8.x86_64.rpm https://mirror.rackspace.com/elrepo/testing/el8/x86_64/RPMS/btrfs-progs-devel-4.20.2-1.el8.x86_64.rpm https://mirror.rackspace.com/elrepo/testing/el8/x86_64/RPMS/btrfs-progs-4.20.2-1.el8.x86_64.rpm https://mirror.rackspace.com/elrepo/testing/el8/x86_64/RPMS/libbtrfs-4.20.2-1.el8.x86_64.rpm  https://mirror.rackspace.com/elrepo/testing/el8/x86_64/RPMS/libbtrfsutil-4.20.2-1.el8.x86_64.rpm http://mirror.centos.org/centos/8/PowerTools/x86_64/os/Packages/device-mapper-devel-1.02.175-5.el8.x86_64.rpm  http://mirror.centos.org/centos/8/PowerTools/x86_64/os/Packages/libassuan-devel-2.5.1-3.el8.x86_64.rpm

Are they not needed anymore?

Reply:
I don't recall installing any package besides those from the script.
I'll verify on a fresh host to confirm nothing else is needed.

Reply:
Thanks!

Reply:
I can confirm no additional package required, simply invoked
```
curl -s -L https://raw.githubusercontent.com/masayag/r4e/k4e-sti/k4e-agent-sti.sh | bash
```